### DISCUSSION TITLE
What is the difference between mmappedChunk and oooNmappedChunk

### DISCUSSION THREAD

**Discussion by Tusenka** [prometheus/prometheus#14998] [General]
mmappedChunk are the chunks from disk. What is the difference between mmappedChunk and oooMmappedChunks? https://github.com/prometheus/prometheus/blob/main/tsdb/head.go#L850 https://github.com/prometheus/prometheus/blob/main/tsdb/head.go#L851

**Comment by bboreham** [comment]
In-order vs out-of-order.
Some description of mmapped chunks: https://ganeshvernekar.com/blog/prometheus-tsdb-mmapping-head-chunks-from-disk
This is an overview of OOO: https://www.youtube.com/watch?v=qYsycK3nTSQ
